src/broadcast_stage.rs

```diff
@@ -52,6 +55,11 @@ fn broadcast(
     inc_new_counter_info!("broadcast_stage-entries_received", num_entries);
 
     let to_blobs_start = Instant::now();
+    *tick_height += ventries
+        .iter()
+        .flatten()
+        .fold(0, |tick_count, entry| tick_count + entry.is_tick() as u64);
```

Comment:
`fold` is a sledgehammer. If a scalpel is available, use that. How about something like?

`flat_map(|entry| entry.is_tick() as u64).sum()`

Reply:
ug, I hate that cast ;)

Reply:
because "entry.is_tick as u64" would also compile...